frame/elections-phragmen/src/lib.rs

```diff
@@ -549,10 +549,13 @@ impl<T: Trait> Module<T> {
 			let members = members_with_stake.into_iter().map(|m| m.0).collect::<Vec<_>>();
 			let result = Ok(maybe_replacement.is_some());
 			let old = [who.clone()];
-			match maybe_replacement {
+			// We don't need to check the actual new members length because it will be the same
+			// as before or one less.
+			// TODO: Is this safe without ensure_can_change_members?
+			let _ = match maybe_replacement {
```

Comment:
```suggestion
			match maybe_replacement {
```

Reply:
I have gotten a compile error for this particular one. Feel free to check out the change and compile yourself :slightly_smiling_face:

Reply:
TIL the compile error was due to a missing semicolon, not the missing `let _ =`